addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -103,6 +110,47 @@ export class WebsiteLoader extends Component {
         });
     }
 
+    /**
+     * Initializes the progress bar.
+     */
+    initProgressBar() {
+        if (this.updateProgressInterval) {
+            return;
+        }
+        // The progress speed decreases as it approaches its limit. This way,
+        // users have the feeling that the website creation progressing is fast
+        // and we prevent them from leaving the page too early (because they
+        // already did XX% of the process).
+        // If there is no module to install, we fake the progress from 0 to 100.
+        // If there is at least 1 module to install, we take 70% of the progress
+        // bar that we divide by the number of modules to install. We fake the
+        // progress of each module individually and when all modules are
+        // installed, we fake the progress of the remaining 30%.
+        const nbModulesToInstall = this.featuresInstallInfo.total || 0;
+        const isSomethingToInstall = nbModulesToInstall > 0;
+        let currentProgress = 0;
+        // This controls the speed of the progress bar.
+        const progressStep = isSomethingToInstall ? 0.04 : 0.02;
+        let progressForAfterModules = isSomethingToInstall ? 30 : 100;
+        let progressForAllModules = 100 - progressForAfterModules;
+        let lastTotalInstalled = 0;
+        let progressPerModule = isSomethingToInstall ?
+            progressForAllModules / nbModulesToInstall : 0;
+
+        this.updateProgressInterval = setInterval(() => {
+            if (this.featuresInstallInfo.nbInstalled !== lastTotalInstalled) {
+                // A module just finished it's install.
+                currentProgress = 0;
+                lastTotalInstalled = this.featuresInstallInfo.nbInstalled;
+            }
+            currentProgress += progressStep;
+            const limit = this.featuresInstallInfo.nbInstalled === nbModulesToInstall ?
+                progressForAfterModules : progressPerModule;
+            this.state.progressPercentage = ((lastTotalInstalled * progressPerModule) +
+                Math.round(Math.atan(currentProgress) / (Math.PI / 2) * limit * 1000) / 1000)
+                .toFixed(2)
```

Comment:
Linter error

Reply:
Also,I guess rounding and toFixed was needed when you showed the percentage -> not needed anymore?